client/src/components/MainPage/TranslationsPage/style.css

```diff
@@ -43,3 +43,8 @@
   display: flex;
   justify-content: flex-end;
 }
+
+.translation__question-text {
+  color: #d11c1c;
```

Comment:
@ali-7 This is just my own opinion and I would trust your design / team lead / product owner over me, but I worry that red text might look like error text to users, and so might be a confusing visual indicator for them.  
I wonder if making the text the original color, but even larger (h1?) might make it stand out more.

Reply:
@tomduggan85 i checked the original color on it but i didn't work with design, but now i checked ``` color: black; ``` actually it looks good with it. i'll try to change it (our p.o. wants red in everything :smiley: ) .
thanks tom :)

Reply:
Haha, sounds good.  Good luck!